inz-emailer/Issues/email/description.xml

```diff
@@ -0,0 +1,70 @@
+<IssueSpecification version="4">
+    <Name>
+        <en>Demo INZ Emailer User e-mail</en>
+        <nl>Demo INZ Emailer User e-mail</nl>
```

Comment:
I will keep Demo INZ Emailer to indicate the name of the issuer and prevent name colliding if we decide to have another issuer issuing e-mail. As we are using INZ as the prefix of our issuers, we will use it as part of the name, in this case, the issuer name is INZ Emailer.

You are right about User. I am going to use Demo INZ Emailer e-mail.

Reply:
Alright.

Please note, however, that the issuer name is already visible for IRMA app users in the bottom of the resulting card in the app. For example, see a screenshot here of our own demo email credential. So currently the `Emailer` part is still double, and it can be removed without causing future name collisions. If you want, it can still be removed later after this PR is merged and deployed.

![Screenshot_20201113-133138_IRMA](https://user-images.githubusercontent.com/979897/99072781-fb932e80-25b4-11eb-932e-dff4e18e4bb7.png)